lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -19,6 +22,8 @@
     'info',
 )
 
+log = logging.getLogger("edx.sneakpeek_middleware")
```

Comment:
This will be a pylint error invalid constant name.  You can disable it with an inline comment.

log = logging.getLogger("edx.sneakpeek_middleware")  # pylint: disable=C0103